angularjs-portal-home/src/main/webapp/staticFeeds/notifications.json

```diff
@@ -41,6 +41,16 @@
         "actionURL" : "/portal/p/post-graduation-plans",
         "actionAlt" : "Access the post-graduation plans survey",
         "priority": false
+      },
+      {
+        "id"     : 7,
+        "groups" : ["Facstaff",
+                    "Facstaff - Former Appointment"],
+        "title" : "2015 WRS Statements of Benefits are posted. Check if you have one.",
+        "actionURL" : "/portal/p/university-staff-benefits-statement",
+        "actionAlt" : "Access the list of available WRS Statement",
```

Comment:
Trivial: Some kind of agreement-in-number problem with this text. Maybe just doesn't matter since the text isn't used for anything?